### ISSUE TITLE
One more videos for class 2.

### ISSUE THREAD

**Issue by kaijieshi7** [NVlabs/dex-ycb-toolkit#35]
I think 20201015-subject-09\20201015_143145 should be class 3. But the video and label is 2. So, the 9th person grasps class 2 for 6 times, and class 3 for 4 times. Is that right?

**Comment by ychao-nvidia** [comment]
That's correct. This was not intended and was a mistake during data collection. However the label has been amended for the mistake so it should still be correct.
